autoload/neomake/makers/ft/solidity.vim

```diff
@@ -9,3 +9,11 @@ function! neomake#makers#ft#solidity#solium() abort
             \ '%f:%l:%c: %t%s: %m',
         \ }
 endfunction
+
+function! neomake#makers#ft#solidity#solhint() abort
+    return {
+        \ 'args': ['-f', 'compact'],
+        \ 'errorformat': '%E%f: line %l\, col %c\, Error - %m,' .
+        \   '%W%f: line %l\, col %c\, Warning - %m,%-G,%-G%*\d problems%#'
+        \ }
```

Comment:
You could use `return neomake#makers#ft#javascript#eslint()` here instead.
This way it is more likely for fixes to go to both.